dts/arm/nxp/nxp_rt1060.dtsi

```diff
@@ -27,7 +27,9 @@
 	};
 };
 
-/* i.MX rt1060 has a second Ethernet controller. */
+/* i.MX rt1060 has a second Ethernet controller.
+ * The FlexCAN3 is capble of CAN FD.
+ */
```

Comment:
These comments seem unrelated. I see no reason for bundling them into one.

Reply:
Fixed this with the latest commit.